charts/spire/templates/spire-controller-manager-configmap.yaml

```diff
@@ -0,0 +1,32 @@
+{{- if .Values.controllerManager.enabled }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "spire.fullname" . }}-controller-manager-config
+  namespace: {{ .Release.Namespace }}
+data:
+  spire-controller-manager-config.yaml: |
+    apiVersion: spire.spiffe.io/v1alpha1
+    kind: ControllerManagerConfig
+    metadata:
+      name: {{ include "spire.fullname" . }}-controller-manager-config
+      namespace: {{ .Release.Namespace }}
+      labels:
+        {{- include "spire.server.labels" . | nindent 4 }}
+    metrics:
+      bindAddress: 127.0.0.1:8082
+    healthProbe:
+      bindAddress: 127.0.0.1:8083
+    leaderElection:
+      leaderElect: true
+      resourceName: 98c9c988.spiffe.io
+      resourceNamespace: {{ .Release.Namespace }}
+    clusterName: {{ .Values.spire.clusterName }}
+    trustDomain: {{ .Values.spire.trustDomain }}
+    ignoreNamespaces:
+      - kube-system
+      - kube-public
+      - spire-system
+      - local-path-storage
```

Comment:
What is this used for?

Reply:
TBH; I don't know

Reply:
Lets remove it. We don't have such namespace.